format_and_overlap_determination.md

```diff
@@ -0,0 +1,123 @@
+
+**Print Serials List**
+: List of Record Ids determined to be serials by some unknown UMich process. List is limited to those held by UMich.
+
+**Large Clusters**
+: Clusters large enough that they create implementation problems. Identified by an OCN, there are currently 11 on the list.
+
+**Billing Entity**
+: The `billing_entity` field taken from the `ht_collections` table associated with a particular HT Item through the collection code. Billing Entity is used for cost allocation when a member's submitted holdings do not include all of their deposited HT Items, or no clustering can be performed due to a lack of OCNs. This also has subtle and likely inconsequential implications for HT Items contributed by UCM and KEIO.
+
+**Collection Code**
+: An [admistrative code](https://www.hathitrust.org/internal_codes) used to share information between Zephir and the Hathitrust repository.
+
+**n_enum**
+: A normalized enumeration. Description fields in the Hathifiles and submitted holdings are processed for enumeration and chronology. Only enumeration ends up being used. 
+
+
+## HT Item Format ##
+Every HT Item has one and only one format. This format is used to calculate the Cluster Format.
+
+**Serial**
+: HT Items with a **Bib Record ID** found on the **Print Serials List**, or in a **Large Cluster** 
+
+**Multi-part Monograph**
+: Is not a Serial. Shares a **Bib Record** with an HT Item that has a non-empty `n_enum`. [^1]
+
+**Single-part Monograph**
+: Is not a Serial or Multi-part
+
+**NB**: A cluster may contain HT Items of multiple formats.  
+**NB2**: Format as reported by members is irrelevant.
+
+## Cluster Format ##
+The Cluster Format is used in the overlap calculations. It is derived from the Item Formats of all the HT Items in the cluster, although it may not and often won't match all of the HT Item formats.
+
+**Multi-part Monograph**
+: Any of the HT Items in the cluster are MPMs.
+
+**SER / SPM**
+: Is not MPM. One or more of the HT Items in the cluster are SPM _and_ one or more of the HT Items in the cluster are SER.
+
+**SER**
+: Is not MPM or SER/SPM. Any of the HT Items in the cluster are SER (and by implication ALL are SER).
+
+**SPM**
+: Is not MPM, SER/SPM, or SER.
+
+## Cost Allocation ##
+
+**Public Domain vs In-Copyright**
+: For cost allocation purposes, HT Items with access "allow" in the Hathifiles are considered "public domain" and access "deny" are considered "in copyright". Rationale for the "allow" and "deny" found in the Hathifiles is underdefined. 
+
+**Target Cost**
+: Set by the Board and Executive Director. Portion of the budget to be paid for by HT Item allocations.
+
+**Cost Per Volume**
+: Target Cost / Total Number of HT Items
+
+**Member Weight**
+: A number assigned to a member by the ED that determines their share of Public Domain costs. 
+
+**Total Weight**
+: The sum of member weights found in the `ht_billing_members` table of the holdings database. **Which process/system has ownership of this table?**
```

Comment:
This probably starts as part of the Estimate Prospective Member Fee process. We need to know member weight when making an estimate. If not then, it ought to happen as part of the Add New Member process. Data entered through OTIS (or, less ideal, manually through mysql).

Reply:
do we need to spell this out more in the documentation itself?

Reply:
I think that's out of scope for this document, but certainly worth recording as part of the documentation of how to do estimates.